.github/workflows/main.yml

```diff
@@ -7,14 +7,19 @@ jobs:
     name: Test
     runs-on: windows-latest
     strategy:
+      fail-fast: false
```

Comment:
Looks like the nightly job was failing due to an ICE in the compiler: https://www.github.com/rust-lang/rust/pull/125493
Looks like they're going to merge soon, so I dont think we need this. Also I'll bypass the nightly CI job when we merge since this is out of our control.